zerolog/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/sillen102/hertz-contrib-zerolog
```

Comment:
module name should be github.com/hertz-contrib/logger/zerolog

Reply:
I missed to change that one after copying from my private repository. Sorry about that. It is fixed now.